project/plugins.sbt

```diff
@@ -9,9 +9,12 @@ val scalaJSVersion =
 libraryDependencies ++= Seq(
   "org.postgresql" % "postgresql"  % "42.2.6", // needed by flyway
   "org.slf4j"      % "slf4j-nop"   % "1.7.28", // to silence some log messages
-  "org.typelevel" %% "cats-core"   % "1.6.1",
-  "org.typelevel" %% "cats-effect" % "0.8"
+  "org.typelevel" %% "cats-core"   % "2.0.0-RC1",
```

Comment:
Could this be the problem? Doesn't this add cats to the sbt classpath?

Reply:
No, that's not a big deal. It was happening before I did that. Also you're logged in as @geminihlswadmin for some reason.